src/Resources/views/fields/content_fields.html.twig

```diff
@@ -3,22 +3,24 @@
 {% extends "EzPublishCoreBundle::content_fields.html.twig" %}
 
 {% block ezobjectrelationlist_field %}
-    {% spaceless %}
+    {% apply spaceless %} %}
```

Comment:
There's an extra `%}` here and in other places below.

Reply:
Good catch, thank you! Follow up PR: https://github.com/ezsystems/ezplatform-http-cache/pull/97